### PR TITLE
✨API_Entity : Move files from API_Ped to API_Entity

### PR DESCRIPTION
**Pull request by dezel-2109**
Hello,
I create this API because some functions like `setHeading` can be also use on vehicles and objects.

### CODE REVIEW

File: src/api/entity/client/setPosition.lua [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    end
+
+    if (keepVehicle) then
+        local vehicle <const> = GetVehiclePedIsIn(entityId, false)
+        if (DoesEntityExist(vehicle) and vehicle ~= 0) then
+            SetPedCoordsKeepVehicle(entityId, coords.xyz)
+            SetEntityHeading(vehicle, coords.w or 0.0)
+            return
+        end
+    end
```
Comment (MoskalykA) [inline src/api/entity/client/setPosition.lua:24]: only ped

**Review by MoskalykA — CHANGES_REQUESTED** [review]

File: src/api/entity/client/setPosition.lua (hunk 1 above)
Comment (dezel-2109) [reply src/api/entity/client/setPosition.lua]: Is it good ?

**Comment by MoskalykA** [comment]
image

**Comment by dezel-2109** [comment]
I'm so dumb sorry

**Review by MoskalykA — APPROVED** [review]

**Pull request merged**